src/rules/enforceSuppressionCode.js

```diff
@@ -0,0 +1,54 @@
+const schema = [
+  {
+    type: 'string',
+  },
+];
+
+const message = (suppression = '') => {
+  return `${suppression} is missing a suppression code`;
+};
+
+const create = (context) => {
+  const isMissingSuppressionCode = function (value) {
+    const suppressionTypes = ['$FlowFixMe', '$FlowExpectedError'];
```

Comment:
What about adding there all the newly supported suppressions? See: https://flow.org/en/docs/config/options/#toc-suppress-comment-regex

> Note: In version v0.127.0, the option to specify the suppression comment syntax was removed. $FlowFixMe, $FlowIssue, $FlowExpectedError, and $FlowIgnore became the only standard suppressions.